contrib/cirrus/runner.sh

```diff
@@ -55,7 +52,8 @@ function _run_validate() {
     bin/podman --version
     bin/podman-remote --version
 
-    make validate  # Some items require a build
+    # Confirm --help and docs match (requires compiled binaries)
+    make man-page-check
```

Comment:
Oh, no. This is a disaster waiting to happen. Please leave this as it was. It's perfectly fine to remove the `Quick checks` step above. This should not be committed under any circumstances as it is. (I really hope I don't need to spend time explaining why).

Reply:
Sorry, I don't understand your concern.  My intention is to split the `make validate` targets across two tasks:

1. "Smoke" which doesn't require compiling anything, just quickly checks code for lint and formatting stuffs.
2. "Validate" which confirms the binary compilation and does more through early testing.

Compiling takes over 5-minutes, so the idea is to fail as early as possible for items that don't require it.  I could see combining the "Validate" and "Smoke" tasks into one, but the cost would be reduced parallelism and possibly longer feedback.